lib/chef/knife/ionoscloud_k8s_create.rb

```diff
@@ -17,6 +17,16 @@ class IonoscloudK8sCreate < Knife
               long: '--version VERSION',
               description: 'The version for the Kubernetes cluster.'
 
+      option :private,
+              long: '--private',
+              default: false,
+              description: 'The indicator if the cluster is public or private. Be aware that setting it to false is currently in beta phase.'
```

Comment:
It seems like the description from here looks like setting the `private` field to `False` is in beta phase, but this means creating a public cluster, not a private one. If we want to add a `--private` tag, it would be useful to clarify the description.

Reply:
done, thank you!